lib/v2/screens/authentication/import_key/interactor/mappers/import_key_state_mapper.dart

```diff
@@ -15,7 +15,9 @@ class ImportKeyStateMapper extends StateMapper {
     if (areAllResultsError(results)) {
       return currentState.copyWith(pageState: PageState.failure, errorMessage: "Error Loading Accounts".i18n);
     } else {
-      List<ProfileModel> profiles = results.map((Result result) => result.asValue!.value as ProfileModel).toList();
+      List<ProfileModel> profiles = results
```

Comment:
is this part of the same issue?

Reply:
is the space needed here?

Reply:
Nope .. what space?

It's a bugfix I found along the way, part of 2.0.1 since without this I can't log in ;) 

Probably not too many people trying to import non seeds accounts but still...

Reply:
Auto formatted this. So should be as it should be.